syft/frameworks/torch/hook/hook.py

```diff
@@ -34,6 +34,13 @@
 
 from syft.exceptions import route_method_exception
 
+from syft.frameworks.crypten import load as crypten_load
+
+try:
+    import crypten
+except e:
+    print("CrypTen not found")
```

Comment:
I think you can use the same strategy than for the imports or PT & TF (but this is not important)